bundles/statistics/statsgrid2016/components/IndicatorParameters.js

```diff
@@ -116,7 +124,19 @@ Oskari.clazz.define('Oskari.statistics.statsgrid.IndicatorParameters', function
 
         me.trigger('indicator.changed', regionsets.length > 0);
     },
-  
+    datasourceSelected: function (datasrc, regionsets) {
+        if (regionsets === null) {
+            return;
+        }
+        return this.paramHandler.testIndicatorsRegionsets(datasrc, regionsets );
+    },
+    regionsetSelected: function (regionsets) {
```

Comment:
Again, not sure why this would be needed? If it really is needed here it should probably be called something like restrictRegionsets()